mailing_list/templates/mailing_list/_mailing_list_details.html

```diff
@@ -0,0 +1,147 @@
+{% load static %}
+
+{% verbatim %}
+<div class="content-section">
+  <div class="float-left col-50">
+    <p>
+      <strong>
+        {{ list.name }}
+        (<a href="{{ ml.listMembersUrl(list) }}"><ng-pluralize
+          count=list.members_count
+          when="{'0': 'No members', 'one': '1 member', 'other': '{} members'}">
+          </ng-pluralize></a>)
+      </strong>
+    </p>
+    <p ng-class="{'caption-disabled': list.access_level == 'readonly'}">
+      <strong>
+        Mailing list address:
+      </strong>
+      {{ list.address }}
+    </p>
+  </div>
+  <div class="float-left col-50 info-btn-status">
+    <a id="email-section-{{ list.section_id }}"
+       href="mailto:{{ list.address }}"
+       class="btn"
+       {% endverbatim %}
+       {% if scope == "class" %}
+         ng-class="list.access_level == 'readonly'
+                   ? 'btn-submit-primary-disabled' : 'btn-submit-primary'"
+       {% else %}
+         ng-class="list.access_level == 'readonly'
+                   ? 'btn-submit-secondary-disabled' : 'btn-submit-secondary'"
+       {% endif %}
+       {% verbatim %}
+       ng-disabled="list.access_level == 'readonly'">
+      <i class="fa fa-envelope"></i>
+      {% endverbatim %}
```

Comment:
This because of mixing Django template tag code (`{% if scope == "class" %}`) with Angular code (`{{ list.name }}`). I'm not a fan of all the sprinkling either. I guess I could use Angular includes, but those are slower, no? Would Angular includes fix up this issue (allow you to pass in a similar additional context to the include and conditionally display things)?